### ISSUE TITLE
[FR]Normalize equity symbols

### ISSUE THREAD

**Issue by yolotrader** [OpenBB-finance/OpenBB#6702]
**What's the problem of not having this feature?**
Currently OpenBB relies on its data providers to resolve a symbol. Unfortunately providers format symbols slightly differently in certain cases. A typical example is that yfinance uses "BRK-A" for Berkshire A shares where as many others use the more common "BRK.A". This requires users to remember these subtleties and developers to write extra code to handle them.

**Describe the solution you would like**
It would be great if OpenBB can adopt a uniform convention for using "." in the symbol format and use the provider extensions to convert it to "-" (or some other character) as necessary. This will help both regular users and developers.

**Describe alternatives you've considered**
If there are other alternatives (worth considering) to your solution, list them here. 

**Additional information**
Add any other information or screenshots about the feature request here.

**Comment by 254Co** [comment]
Would be worthwhile to integrate OpenFGI API. 

https://www.openfigi.com/api

**Comment by deeleeramone** [comment]
Yes, however, it would be for a different purpose. OpenFIGI won't help with mapping between vendor's symbology, and especially where a ticker symbol is the only type of input allowed. 

Is it `BRK.A` or `BRK-A` or `BRK.A:US` or  `BRK/A`? Nuances there are very specific to vendors and most don't have handling for different formats.
